twitchpopups.htm

```diff
@@ -21,32 +22,37 @@
     </script>
 
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <script src="jquery-3.4.1.min.js"></script>

```

Comment:
I think this is subjective. Nothing wrong with jQuery.

Reply:
![firefox_sDiYSljycS](https://user-images.githubusercontent.com/226692/82113204-9c846680-9797-11ea-865a-39470539cad6.png)

I meant nothing by it, just removing unused code 👀